src/blocks/file-blocks/geojson.tsx

```diff
@@ -0,0 +1,335 @@
+                              const isCollection =
+                                newGeojson.type === "FeatureCollection";
+                              if (isCollection) {
+                                newGeojson.features[index].properties[key] =
+                                  value;
+                                newGeojson.features = newGeojson.features.map(
+                                  (d = {}, i) => {
+                                    if (!d) return;
+                                    const {
+                                      __geojson_internal_index__,
+                                      ...properties
+                                    } = d.properties || {};
+                                    return { ...d, properties };
+                                  }
+                                );
+                              } else {
+                                const {
+                                  __geojson_internal_index__,
+                                  ...properties
+                                } = newGeojson.properties || {};
+                                newGeojson.properties = {
+                                  ...properties,
+                                  [key]: value,
+                                };
+                              }
+                              setModifiedContent(JSON.stringify(newGeojson));
+                              setIsDirty(true);
+                            }}
+                          />
+                        ) : (
+                          <div className="py-1 px-2">
+                            {JSON.stringify(hoveredFeature.values_[key])}
+                          </div>
+                        )}
+                      </div>
+                    </Fragment>
+                  );
+                }
+              )}
+          </div>
+        </div>
+      )}
+
+      {isDirty && (
+        <button
+          className="position-absolute d-block top-2 right-2 btn btn-primary"
+          onClick={() => {
+            const contentString = modifiedContent;
+            onRequestUpdateContent(contentString);
+          }}
+        >
+          Save changes
+        </button>
+      )}
+    </div>
+  );
+}
+
+const parseGeoJSON = (geojson: any) => {
+  try {
+    return new GeoJSON({
+      featureProjection: "EPSG:3857",
+    }).readFeatures(geojson);
+  } catch (e) {
+    console.log(e);
+    return [];
+  }
+};
+
+const EditableText = ({
+  value,
+  onChange,
+}: {
+  value: string;
+  onChange: (value: string) => void;
+}) => {
+  const [isEditing, setIsEditing] = useState(false);
+  const [inputValue, setInputValue] = useState(value);
+  const onChangeInput = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setInputValue(e.target.value);
+  };
+  const inputElement = useRef<HTMLInputElement>(null);
+  useEffect(() => {
+    if (isEditing) {
+      inputElement.current?.focus();
```

Comment:
I wonder if you get this line for free with the `autofocus` attribute on the input